scripts/detect-english-leaks.sh

```diff
@@ -9,6 +9,11 @@ export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
 echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
 echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
 
+if ! setup_service node v16.18.1 &> /dev/null; then
+  echo "Node 16 is required to run English leak tests"
+  exit ${FAILED_SETUP}
+fi
```

Comment:
We don't need the conditional and extra block here if we fail the entire script if tand command exits non-zero. You can do this by setting a flag at the top of the bash script:

```
set -e
```